src/platform_impl/macos/event_loop.rs

```diff
@@ -114,8 +120,23 @@ pub struct EventLoop<T: 'static> {
     _callback: Option<Rc<RefCell<dyn FnMut(Event<'_, T>, &RootWindowTarget<T>, &mut ControlFlow)>>>,
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Hash)]
+pub(crate) struct PlatformSpecificEventLoopAttributes {
```

Comment:
The entire `platform_impl` module is non-public so if I understand correctly, then everything within it can be just `pub` and it will only be visible to the crate. It would probably look cleaner that way.

https://github.com/rust-windowing/winit/blob/001fb7ef60efe989b54b07024b660fd07e94afc3/src/lib.rs#L158

Reply:
You are making a good point. I'll leave it up to you.

Reply:
I'll leave it as is for now at least, because I prefer the explicitness